packages/sdk-vue/src/FusionAuthVuePlugin.ts

```diff
@@ -1,10 +1,11 @@
-import { App } from 'vue';
+import { App, InjectionKey } from 'vue';
 import { FusionAuth, FusionAuthConfig } from './types.ts';
 import * as components from './components/index.ts';
 import { fusionAuthKey } from './injectionSymbols.ts';
 import { createFusionAuth } from './createFusionAuth/index.ts';
+import { UserInfo } from '@fusionauth-sdk/core';
```

Comment:
This should be `UserInfo` from `./types.ts`

Reply:
https://github.com/FusionAuth/fusionauth-javascript-sdk/issues/147
Separate issue, though. Here's a ticket https://github.com/FusionAuth/fusionauth-javascript-sdk/issues/147